shop/flywayConfig.conf

```diff
@@ -0,0 +1,5 @@
+flyway.locations=filesystem:db/migrations
+flyway.url=jdbc:h2:~/Documents/Code/online-shop-LechintanTudor/shop

```

Comment:
Let's only use project related paths. 
-> the project should be cloned and started locally by anyone and not everyone has the file Code in Documents 😄